src/sections/ServicesHero/index.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import Section from "root/components/Section";
+import Typography from "root/components/Typography";
+import BackgroundVideoText from "root/components/BackgroundVideoText";
+
+import "./index.css";
+
+const column = 114;
+const gutter = 28;
+const eightColumns = 8 * column + 7 * gutter;
+const sixColumns = 6 * column + 5 * gutter;
+const breakpointMobile = 768;
+const breakpointDesktop = 1268;
+
+const renderCopy = () => (
+  <div>
+    <div styleName="left">
+      <Typography variant="h2" color="dark-blue" fontFamily="meta-serif">
+        Working together to achieve your goals
+      </Typography>
+    </div>
+    <div styleName="right">
+      <Typography color="dark-blue">
+        We’ll work alongside you from the early stages of product research and
+        design to development and deployment, all while continuously delivering
+        you product updates.
+      </Typography>
+    </div>
+  </div>
+);
+
+const renderDesktop = () => (
+  <Section>
+    <BackgroundVideoText
+      key="desktop"
+      labels={["A dynamic team to design", "and develop your product"]}
+      poster="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/oceanshot.jpg"
+      video="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/ocean-small.webm"
+      height="195px"
+      width={`${eightColumns}px`}
+    />
+    {renderCopy()}
+  </Section>
+);
+
+const renderMobile = () => (
+  <Section>
+    <div>Implement Mobile</div>
+    {renderCopy()}
+  </Section>
+);
+
+const renderTablet = () => (
+  <Section>
+    <BackgroundVideoText
+      key="tablet"
+      labels={["A dynamic team to", "design and develop", "your product"]}
+      poster="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/oceanshot.jpg"
+      video="https://s3-us-west-2.amazonaws.com/s.cdpn.io/4273/ocean-small.webm"
+      height="330px"
+      width={`${sixColumns}px`}
+    />
+    {renderCopy()}
+  </Section>
+);
+
+const ServicesHero = () => {
+  const width = document.documentElement.clientWidth;
```

Comment:
And that's why the build is failing because `document` is undefined when Gatsby is trying to build the pages `WebpackError: ReferenceError: document is not defined`

Reply:
This component should do 3 things, first it should render based on a `jsReady` state attribute. If `jsReady` then render our thing, if not ready just render `null`. `jsReady` is false by default and then you `setState({jsReady: true})` on the `componentDidMount` method, because this method is only called in the browser and not on the server-side render step of Gatsby build.

Finally, on top of the component we should have a `noscript` tag, that inside has a regular `Typography` component just like in the hero homepage. We need the `noscript` otherwise nothing will happen for browsers with javascript disabled.

Reply:
Ok, thanks. Nice explanation 👌

Reply:
What about the `noscript`?